SQRLDotNetClientUI/Program.cs

```diff
@@ -58,10 +61,14 @@ public static void Main(string[] args)
                         hasHandle = true;
                     }
 
+                    // Perform database migrations
+                    SQRLDBContext _db = new SQRLDBContext();
+                    _db.Database.Migrate();
```

Comment:
Very nice work on this, I was getting concerned about the migration bit! as I was reading the changes.

Reply:
Yeah, me too! I had no prior experience with EF and the whole migrations topic, and I was/am kind of worried that stuff might break out in the field. When developing, we run all the fancy command line tools to setup and perform the migrations, but how is it all working for a normal user?

I learned that without the call to `DatabaseContext.Database.Migrate()`, stuff would break, so I read the docs and after adding this call everything started working on the installations that had "old" DBs.

I'm still a little sceptic about all of this, since so much is happening implicitly behind the scenes, but it seems that everything is working so far. Hope your tests go well, too!